docs/upgrade.html

```diff
@@ -20,6 +20,45 @@
 <script id="upgrade-template" type="text/x-handlebars-template">
 
 <h4><a id="upgrade_2_1_0" href="#upgrade_2_1_0">Upgrading from 0.8.x, 0.9.x, 0.10.0.x, 0.10.1.x, 0.10.2.x, 0.11.0.x, 1.0.x, 1.1.x, or 2.0.0 to 2.1.0</a></h4>
+
+<p><b>Note that 2.1.x contains an incompatible change to the internal schema used to store consumer offsets. Once the upgrade is
```

Comment:
It seems that we will have the issue of not being able to downgrade as long as there is change to the schema. Do you need the word `incompatible` here? I am not sure what we additionally mean by `incompatible` change as compared to `change`.

Reply:
Yeah, agreed. It makes it sound scarier than it is.